modules/apps/application-list/application-list-user-personal-site-permissions/src/main/java/com/liferay/application/list/user/personal/site/permissions/internal/model/listener/CompanyModelListener.java

```diff
@@ -57,6 +76,32 @@ public void onAfterCreate(Company company) {
 			});
 	}
 
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		_userPersonalSitePermissions = new UserPersonalSitePermissions();
+
+		String filter = StringBundler.concat(
```

Comment:
This is not right, with this change you will end up with filter having duplicated content.
```"(&(&(objectClass=com.liferay.application.list.PanelApp)(panel.category.key=site_administration*))(objectClass=com.liferay.application.list.PanelApp))"```

Also don't do inner class in another inner class, that's just unnecessary structure complexity.